.github/workflows/update-docker-requirements.yml

```
@@ -1,11 +1,9 @@
 name: docker-requirements
 
 on:
-  pull_request:
-    paths:
-      - ".github/workflows/update-docker-requirements.yml"
-      - "pyproject.toml"
-      - "requirements-docker.lock"
+  push:
+    branches:
+      - main
```

```suggestion
      - main
    paths:
      - ".github/workflows/update-docker-requirements.yml"
      - "pyproject.toml"
      - "requirements-docker.lock"
```

If you'd only like to run this if these files are changes, feel free to commit this suggestion.